proposals/lip-0061.md

```diff
@@ -520,10 +523,10 @@ def selectAggregateCommit() -> AggregateCommit:
     # Note that the BFT parameters at height maxHeightCertified+1 are already authenticated by the previous certificate.
     heightNextBFTParameters = getNextHeightBFTParameters(getBFTHeights().maxHeightCertified + 1)
     if previous function call returns valid height:
-        nextHeight = min(heightNextBFTParameters-1, getBFTHeights().maxHeightPrecommitted)
+        nextHeight = min(max(heightNextBFTParameters -1, MIN_CERTIFICATE_HEIGHT), getBFTHeights().maxHeightPrecommitted)
```

Comment:
question: below we basically use the `max(heightNextBFTParameters, MIN_CERTIFICATE_HEIGHT)`, here instead we have an extra -1, what is the reason?

Reply:
`nextHeight` should be the highest height for which we attempt to create an aggregate commit. What should be the value of `nextHeight`? `heightNextBFTParameters -1`, because if we have BFT params at a height `h`, we want the certificate at height `h-1`. But in case this value is smaller than `MIN_CERTIFY_HEIGHT`, we can not create such a certificate, so set it to `MIN_CERTIFY_HEIGHT`